lively.ide/studio/controls/text.cp.js

```diff
@@ -116,16 +116,24 @@ export class RichTextControlModel extends ViewModel {
           italicStyle, underlineStyle, quote,
           lineWrappingSelector, paddingControls
         } = this.ui;
+
         const { activeButtonComponent, hoveredButtonComponent } = this;
-        this.models.fontFamilySelector.items = availableFonts().map(font => {
+
+        const fontItemCreator = font => {
           return {
             value: font,
             string: font.name,
             isListItem: true
           };
-        });
-
-        fontFamilySelector.selection = text.fontFamily.replace(/^"(.*)"$/, '$1');
+        };
+        const projectFontItems = $world.openedProject.projectFonts.map(fontItemCreator);
```

Comment:
This crashes when no projectFonts or openedProject is present.

Reply:
Should be resolved!